exercises/bank-account/bank_account_test.go

```diff
@@ -296,6 +296,31 @@ func TestConcDeposit(t *testing.T) {
 	}
 }
 
+// The benchmark operations are here to allow you to try different
```

Comment:
Taking another look I think "encourage" is the right word. 

(I appreciate the feedback, more feedback == better commits!)